lib/bolt/result.rb

```diff
@@ -121,9 +125,7 @@ def status_hash
     end
 
     def generic_value
-      if @value_set
```

Comment:
This method was only used in one place, so it seemed weird to return a `nil` value, especially since we might no longer display empty JSON objects.

Reply:
We can get rid of the `@value_set` variable then